ui/webui/src/components/Error.jsx

```diff
@@ -83,6 +98,7 @@ export const BZReportModal = ({
     }, []);
 
     const openBZIssue = (reportURL, logFile, logContent) => {
+        reportURL = ensureMaximumReportURLLength(reportURL);
         reportURL = addLogAttachmentCommentToReportURL(reportURL, logFile);
```

Comment:
Is this the right order? First you trim it and then add the comment.

Reply:
Yes it is. There number is taking the added comment (of ~ static length) into account. We want the added comment to be there always, irrespective of the message being cut. I'll explain it in a comment in the code as well.